docs/authentication/overview.mdx

```diff
@@ -1,5 +1,64 @@
 ---
-sanity_slug: /docs/authentication/overview
+title: Sign-up & Sign-in
+description: Learn how to configure authentication and user management for your Clerk application.
 ---
 
-# This is a stub
+# Sign-up & Sign-in
+
+Clerk supports multiple authentication strategies so that you can implement the strategy that makes sense for *your* users. You can use Clerk's [hosted pages](/docs/authentication/using-clerk-hosted-pages), [pre-built components](/docs/authentication/components/overview), or [build your own custom flows](/docs/custom-flows/overview).
+
+## Configuration
+
+Clerk's configuration settings affect how the users of your application can [sign up](/docs/authentication/configuration/sign-up-options) and [sign in](/docs/authentication/configuration/sign-in-options) and which attributes are editable via their user profile. You can also manage [user sessions](/docs/authentication/configuration/session-options), [control who gets access](/docs/authentication/configuration/allowlist) to your application, and [customize the email & SMS messages](/docs/authentication/configuration/email-options) that are sent by Clerk during authentication flows. All of these settings can be found under the **User & Authentication** section in your [Clerk Dashboard](https://dashboard.clerk.dev/).
+
+Check out our detailed guide for information on [how to set up your application](/docs/authentication/set-up-your-application).
+
+## Custom Flows
+
+Clerk provides the flexibility to create [custom sign-up and sign-in flows](/docs/custom-flows/overview) for your application. Below are some of the most common flows that you can implement with Clerk.
+
+[Email & Password](/docs/custom-flows/email-password)
+
+With this setup, users that come to your website will need to supply an authorized email and password to gain access to their account.
+
+[Email / SMS OTP](/docs/custom-flows/email-sms-otp)
+
+During sign-in, users will enter their identifier (email address or phone number) to receive a one-time code and complete the authentication process.
+
+[Magic Links](/docs/custom-flows/magic-links)
+
+During login or registration, users will enter their email address to receive an email message with a link that can be clicked and complete the authentication process.
+
+[Multifactor](/docs/custom-flows/mfa)
+
+By enabling multifactor authentication (MFA), often referred to as two-factor authentication or 2FA, you can encourage or require your users to perform a second verification check during sign-in.
+
+[Social Login Providers](/docs/authentication/social-connections/overview)
+
+Clerk offers a list of Social Login providers that can be used during sign-up and sign-in. There will be a button on the sign-in and sign-up pages with the provider's logo and the appropriate text for each of the selected providers.
+
+Don't see the provider you need? Request others [here](https://clerk.com/support); we can usually add them within a week.
```

Comment:
I'm not even sure if this is necessary here. I feel this would have been better on the custom-flows/overview 
But these topics are not talked about in /docs/authentication . I think the reference to /custom-flows in the introductory paragraph (below our H1) is enough.

Reply:
I kind of agree 👍 Linking to the custom flows overview seems reasonable here.